### DISCUSSION TITLE
First Timer - Need Help Setting Up

### DISCUSSION THREAD

**Discussion by BGillham** [HotcakesCommerce/hotcakes-commerce-core#462] [Q&A] [answered]
I am a complete beginner to Hotcakes but have used DNN for several years. Is there any easy startup procedure that I can follow to get going in my "Test" Environment?
eg:
1. Install DNN 9.12.x (Done)
2. Install Hotcakes as a Module (Have the Source but where is the Install?)
3. ???
Please assist - I would love to get going with Hotcakes...
I have tried the https://mmmcommerce.com/Resources/Downloads but the site is no longer there!

**Comment by Orrouk** [comment]
I understand. The first time, whatever it is, is never obvious. Especially with HCC, I can say, I have it install on Or-rouge.com.
Did you tried to use the source as install files? Can't remember everything but if I faced the same situation, I certainly tried cause it's what I would do now. 
Is it possible to install HCC in DNN directly from it's repository on GitHub ? Should we fork the HCC's repository before so we can protect our customisations when upgrading the same way in the futur ?

**Reply by BGillham** [reply]
I tried to install the Source.zip as an "Extension" but that failed... I have no idea what to do next!

**Reply by Orrouk** [reply]
I guess I download the package (hotcakes-commerce-core-development) from github. The install files are in the folder WebDeploy... as you show in the image. 
![image](https://github.com/HotcakesCommerce/hotcakes-commerce-core/assets/8306168/a75478cc-366f-414b-b2c5-a0e3ea42fd1d)
Once installed, HCC is pretty easy to configure. Everything else should go straight forward and the documentation produces by Will Stroll on his website is quite complete and accurate. I found this ref in my blog: https://hotcakescommerce.zendesk.com/hc/en-us/ 

Just remember 3 things that may spare your time or your life, make the difference, drive right througt success instead of fuckoying for nothing. 

You will not be able to import products from excel if more than one language are enable in the portal. The "Import" button will not even be there anymore. It will reappeared if you turn off the multi-language fonction... Whatever the language you choose to stay with when turning off the "multi factor", the products will always be shown to webusers on the site, but they will not be manageable by import/export if you are not in the original language you used when you created the store. 

Also, in HCC, there are 2 places where you can configure a connection for one or another type of PayPal account and there is 3 types. With Authorized, you don't need the help of a reseller, just go on their website and use the chat support. 

Reach me if you face any other difficulties, I'll help you.

**Reply by BGillham** [reply]
Tried...
I setup a new DNN v9.12.00
I downloaded the Source for HCC v3.07.00
I tried to install the HCC.Application_03.03.00_Install.zip as a Module from the WebDeploy Folder
!!! Everything CRASHES !!!
What I am doing wrong? Am I just being stupid?
Is there a Step-by-Step procedure that I can follow to get this right?
Then I tried...
Copying the Website folder to my in-house website and that just crashes...
Sorry folks... I am very confused!

**Reply by erw13n** [reply]
@BGillham The https://mmmcommerce.com back online, try download the installation package there.

**Reply by erw13n** [reply]
And try follow this Installation tutorial https://hotcakescommerce.zendesk.com/hc/en-us/articles/208028996-Installation-Cheat-Sheet

**Comment by WillStrohl** [comment, marked as answer]
It sounds like you're not using the correct installation package.  The correct name of the installation package is something like the following format:  

HotcakesCommerce_03.07.00_Core_Install.zip

If version 3.7.1 were to be released at some point, that package name would only change in version number, such as the following example:  

HotcakesCommerce_03.07.01_Core_Install.zip  

Just to further the example so any future readers can also benefit, if the version were to be increased to 3.8.0 or 4.0, those examples are below.  

HotcakesCommerce_03.08.00_Core_Install.zip
HotcakesCommerce_04.00.00_Core_Install.zip

With that being said, there are indeed other zip packages that all have specific purposes, but the primary one for nearly all use cases is what we're mentioning here.  

**Additional Background**  
`Core` refers to the core module.  This is a legacy naming convention from when we were also deploying a CMS version of the module that included the module and DNN pre-integrated as a single solution.
